test/acceptance/service-fabrik-admin.internal-db.lifecycle.spec.js

```diff
@@ -63,6 +64,7 @@ describe('service-fabrik-admin', function () {
       it('should provision service fabrik internal mongodb when deployment not found', function (done) {
         const WAIT_TIME_FOR_ASYNCH_CREATE_DEPLOYMENT_OPERATION = 200;
         this.timeout(2000 + WAIT_TIME_FOR_ASYNCH_CREATE_DEPLOYMENT_OPERATION);
+        mocks.director.getBindingProperty(CONST.FABRIK_INTERNAL_MONGO_DB.BINDING_ID, {}, config.mongodb.deployment_name, 'NOTFOUND');
```

Comment:
'NOTFOUND' -->same as above.

Reply:
Thats the name of the error class, no need to define it in constants.